packages/cli/templates/ui/package.json

```diff
@@ -41,6 +43,7 @@
     "@angular/language-service": "^6.1.0",
     "@types/jasmine": "2.5.47",
     "@types/node": "7.0.12",
+    "@vcd/ext-cli": "~0.0.1",
```

Comment:
The version doesn't match the new version of the cli package, also I think it's better to use exact versions in order to avoid further confusions (when a bug appear).

Reply:
I think it should be fine for development, but when we release we should use exact versions. Right now we are not releasing, but actively developing and its a bit of pain to go and update all dependency versions.